src/models/Forecast.js

```diff
@@ -0,0 +1,105 @@
+export default class Forecast {
+  constructor(data) {
+    this.data = data;
+  }
+
+  createForecast() {
+    const forecastSection = document.querySelector('.forecast');
+    forecastSection.innerHTML = '';
+
+    Object.values(this.data).map((dayForecast, index) => {
+      const date = new Date(dayForecast.datetime);
+
+      const forecastBlockHtml = `
+        <div class="forecast-block">
+          <div class="forecast-block__content">
+            <h3 class="forecast-block__date">
+              ${this.getWeekDay(date)} ${date.getDate()}
+            </h3>
+            <div class="forecast-block__icon-container">
+              <img class="forecast-block__icon">
+            </div>
+            <span class="forecast-block__temp">
+              ${dayForecast.min_temp}°C - ${dayForecast.max_temp}°C
+            </span>
+          </div>
+        </div>
+      `;
+      forecastSection.insertAdjacentHTML('beforeend', forecastBlockHtml);
+
+      const forecastIcon = forecastSection.querySelectorAll('.forecast-block__icon')[index];
+      forecastIcon.src = require(`../images/weather-icons/${dayForecast.weather.icon}.png`);
+
+      const forecastBlock = forecastSection.querySelectorAll('.forecast-block')[index];
+
+      forecastBlock.addEventListener('click', () => {
+        this.addActiveContent(forecastBlock);
+        this.createForecastCard(dayForecast, date);
+      })
+
+      if (index === 0) {
+        this.addActiveContent(forecastBlock);
+        this.createForecastCard(dayForecast, date);
+      }
+    });
+  }
+
+  addActiveContent(forecastBlock) {
+    const allforecastContent = document.querySelectorAll('.forecast-block__content');
+
+    allforecastContent.forEach(content => {
+      content.classList.remove('forecast-block__content--active');
+    })
```

Comment:
Select only active, not all of them
```suggestion
    document.querySelectorAll('.forecast-block__content--active').forEach(content => {
      content.classList.remove('forecast-block__content--active');
    })
```